roles/pulp3/README.md

```diff
@@ -37,6 +37,8 @@ Role Variables:
   the systemd module's "state" directive. Defaults to "started".
 * `pulp_wsgi_enabled` This variable can be configured with any of the states allowed
   by the systemd module's "enabled" directive. Defaults to "true."
+* `pulp_port` Set the port the API server is served from. Defaults to '8000'.
```

Comment:
After taking a second look, could you change this to `pulp_api_port`, `pulp_api_host`